packages/design-system-scripts/gulp/docs/docs.js

```diff
@@ -85,6 +86,7 @@ module.exports = {
     await cleanDist(docsPackageDir);
     await extractReactDocs(sourcePackageDirs, options.rootPath);
     await generatePages(sourcePackageDirs, docsPackageDir, reactDataPath, options);
+    await runWebpackStatically(sourcePackageDir, docsPackageDir, options.rootPath);
```

Comment:
@pwolfert Was this what you had in mind for running webpack statically? I also have to call `compileDocsSass` after this right?

https://github.com/CMSgov/design-system/pull/659/commits/9a28dabce8364f050b64afea61167ac737a312b2

Reply:
Yeah, that looks great! Yes about compiling sass; the original did it in this order: https://github.com/CMSgov/design-system/blob/master/tools/gulp/build.js#L95